### PR TITLE
Fixed: Fix the issue with the automation names in the table.

### PR DESCRIPTION
**Pull request by fit2bot**
Fixed: Fix the issue with the automation names in the table.

### CODE REVIEW

**Review by ZhaoJiSen — APPROVED** [review]

File: src/views/accounts/Automation/BaseExecutionList.vue [hunk 1]
```diff
@@ -87,8 +87,8 @@ export default {
             formatter: DetailFormatter,
             minWidth: '180px',
             formatterArgs: {
-              getTitle: ({ row }) => row.snapshot.name,
-              getDrawerTitle: ({ row }) => row.snapshot.name,
+              getTitle: ({ row }) => row.automation?.name || row.snapshot.name,
+              getDrawerTitle: ({ row }) => row.automation?.name || row.snapshot.name,
               getRoute: ({ row }) => ({
                 name: this.automationRoute,
                 params: { id: row.automation.id }
```
Comment (BaiJiangJie) [inline src/views/accounts/Automation/BaseExecutionList.vue:94]: As of 2023/07/16, there are no known issues with the provided JavaScript code snippet related to programming errors or optimizations. It looks like it is syntactically correct and functional according to its usage.

For better readability:

```javascript
export const detailFormatter = (row) => {
  if(row.automation) row.automation.name ? () => (formatter)(row) : function () {};
}

export default {
    title: 'DetailFormatters',
     formatterArgs: {
         getNameFromRow: (... args) => row.snapshot,
      },
}
```

Also, it's always good practice to have comments explaining the logic behind each variable/function when using them for future reference, debugging purposes, or maintaining the codebase over time:

`${
this.automationId ||
   console.error("Automation ID missing! Please call loadData on Automation component.")
}`

This checks that `id` from the row object has been passed into the `getRoute` method correctly. Always test edge cases before implementing such conditional expressions in production environments where user experience may be compromised.
